XenModel/XenAPI-Extensions/Address.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace XenAPI
+{
+    public abstract class Address

```

Comment:
Why is this class abstract? Did you rather mean static?

Reply:
You are correct, I meant abstract. I will change it.